### PR TITLE
Prepare web 21.04.8

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,53 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] Macro as Password not working with snmpv3 extraoptions
+- [Configuration] Massive changes on hosts reset untouched values
+- [Configuration] Unwanted empty string parameter for LUA output in Broker configuration
+- [Custom Views] Can't share views with LDAP groups
+- [Downtime] Impossible to set a downtime with french GUI
+- [Downtime] Planned downtimes are shown if one downtime is started
+- [Downtime] fixed PHP fatal error when saving a downtime on a hostgroup
+- [Graph] Fixed wrong scale and metric value with negative values
+- [Knowledge Base] Get correct link from template. Based on [PR #10066](https://github.com/centreon/centreon/pull/10066)
+- [Resources Status] Fixed wierd behaviord of the ressources status page when applying filters and ordering by a specific column.
```
Comment (mdebauchez) [inline en/releases/centreon-core.md:49]: 'weird behaviors' instead of 'wierd behaviord' I think

File: fr/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,37 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] Macro as Password not working with snmpv3 extraoptions
+- [Configuration] Massive changes on hosts reset untouched values
+- [Configuration] Unwanted empty string parameter for LUA output in Broker configuration
+- [Custom Views] Can't share views with LDAP groups
+- [Downtime] Impossible to set a downtime with french GUI
+- [Downtime] Planned downtimes are shown if one downtime is started
+- [Downtime] fixed PHP fatal error when saving a downtime on a hostgroup
+- [Graph] Fixed wrong scale and metric value with negative values
+- [Knowledge Base] Get correct link from template. Based on [PR #10066](https://github.com/centreon/centreon/pull/10066)
+- [Resources Status] Fixed wierd behaviord of the ressources status page when applying filters and ordering by a specific column.
```
Comment (mdebauchez) [inline fr/releases/centreon-core.md:49]: Same comment as in english version

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,53 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Resources Status] Performance chart not displayed reliably on firefox
+- [UX] Do not display orphan menus in UI
+
```
Comment (mdebauchez) [inline en/releases/centreon-core.md:52]: We don't mention this ticket ? 
https://centreon.atlassian.net/browse/MON-11382

**Review by mdebauchez — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md [hunk 4]
```diff
@@ -19,6 +19,53 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.04.8
+
```
Comment (tuntoja) [inline en/releases/centreon-core.md:23]:
```suggestion

`Release date: November 15, 2021`

```

File: fr/releases/centreon-core.md [hunk 5]
```diff
@@ -19,6 +19,37 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.04.8
+
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:23]:
```suggestion

`Release date: 15 novembre 2021`

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

**Review by tuntoja — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 6]
```diff
@@ -19,6 +19,55 @@ If you have feature requests or want to report a bug, please go to our
...
+`Release date: November 15, 2021`
```
Comment (tuntoja) [inline en/releases/centreon-core.md:26]:
```suggestion
```

File: fr/releases/centreon-core.md [hunk 7]
```diff
@@ -19,6 +19,39 @@ If you have feature requests or want to report a bug, please go to our
...
+`Release date: 15 novembre 2021`
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:26]:
```suggestion
```

**Review by tuntoja — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**